### Footstep parameterization for the Aldebaran Robotics Nao humanoid #########

### footstep settings #########################################################
# Footstep displacement vectors (in meter / rad)
# WARNING: data type has to be float since integers will cause an exception
footsteps:
  x:     [0.00, 0.08,-0.04, 0.00, 0.05, 0.01, 0.015, 0.04,-0.03, 0.06, 0.04,-0.02]
  y:     [0.16, 0.09, 0.09, 0.12, 0.14, 0.13, 0.100, 0.12, 0.12, 0.12, 0.10, 0.12]
  theta: [0.00, 0.00, 0.00, 0.00, 0.00,-0.50, 0.500, 0.30, 0.50, 0.00, 0.00, 0.00]


### foot settings #############################################################
foot:
  size: {x: 0.16, y: 0.088, z: 0.015}
  separation: 0.095
  origin_shift: {x: 0.02, y: 0.006}
  max:
    step: {x: 0.08, y: 0.16, theta: 0.3}
    inverse:
      step: {x: -0.04, y: 0.09, theta: -0.3}


### step range ################################################################

# more accurate than the bounding box defined by the max/max inverse step
step_range:
  x: [0.00, 0.01, 0.02, 0.03, 0.05, 0.06, 0.07, 0.07,-0.03,-0.03,-0.02,-0.01]
  y: [0.15, 0.15, 0.15, 0.14, 0.13, 0.13, 0.12, 0.09, 0.09, 0.13, 0.14, 0.15]
